### DISCUSSION TITLE
Clusterscoped messaging topology operator access to rabbitmq cluster

### DISCUSSION THREAD

**Discussion by ilabrovic** [rabbitmq/messaging-topology-operator#833] [Q&A]
The messaging topology operator needs access to port 15672 on each rabbitmq cluster.
To get this working we need to add networkpolicies in each namespace that has a rabbitmq cluster to allow traffic from the namespaces where the operator is installed (openshift-operators).

Assuming the operator just wants to verify if the port is alive, can this check somehow be disabled?

**Comment by Zerpet** [comment]
The Topology Operator needs access to the management API port 15672 to perform all its operation. When you create a custom resource `Queue`, the Topology Operator sends an HTTP request to RabbitMQ HTTP API to create a Queue. The same applies for other custom resources. I understand that this can be inconvenient with network policies in place. I'm not experienced with network policies in Kubernetes, any suggestions to improve this pain point are welcome (other than not using rabbit HTTP api)
